### ISSUE TITLE
Redux via jspm bigger than including via cdn

### ISSUE THREAD

**Issue by babakness** [jspm/jspm-cli#2017]
Hello,

When I include Redux into my project which was install via `jspm install redux` my bundle size increases by 3kb gzipped than if I were to save it locally from a CDN provider and then `import` directly.

https://cdnjs.cloudflare.com/ajax/libs/redux/3.5.2/redux.min.js

In both cases they are minified before gzipped.

This is also the case with other libraries like BlissJS. In the case of bliss I install from npm.

Jspm version 0.16

**Comment by guybedford** [comment]
Did you try using jspm 0.17 here?

**Comment by guybedford** [comment]
jspm 0.17 should have a better footprint I think as it deals with the conditional requires better.

**Comment by babakness** [comment]
Hi @guybedford , I upgraded to 0.17@beta. I have tried redux with it but did give `blissfuljs` a go. I install as follows

`jspm install npm:blissfuljs`

I bundle using

`jspm bundle main.js`

I also tried this way

`jspm build main.js bundle.js`

This increased the bundle size by ~20kb. Compared to the same app bundled with webpack@beta, the jspm version increased by over 10kb more.

With the `jspm build` method I get a notice about optimization but not with the `jspm bundle` method; both are roughly the same size.

Not sure if I'm bundling / building correctly with `jspm`. Any advise appreciated, thanks.
